scripts/check-links.sh

```diff
@@ -49,4 +51,14 @@ retry() {
     done
 }
 
+# Start an HTTP server listening at the root of the built website.
+yarn run http-server public --port $HTTP_SERVER_PORT &>/dev/null &
+HTTP_SERVER_PID=$!
+
+# Run the link checker once the HTTP server is listening.
+while ! nc -z localhost $HTTP_SERVER_PORT; do sleep 0.1; done
+# Re-run the link checker up to three times, to handle the occasional transient failure.
 retry check_links
+
+# Kill the server process.
+kill -f $HTTP_SERVER_PID
```

Comment:
Is the `-f` flag needed for `kill`?

Reply:
Ah, no, my oversight -- it isn't. I'll remove it.